vector/src/main/res/layout/view_current_session.xml

```diff
@@ -21,15 +21,15 @@
         tools:src="@drawable/ic_device_type_mobile" />
 
     <TextView
```

Comment:
The size of the view with id `currentSessionVerificationStatusDetailTextView` seems too small compared to Figma. Can you check? I have started a `styles_devices_management.xml` file if needed.

Reply:
Right, fixed.